conmon-rs/server/src/child_reaper.rs

```diff
@@ -109,6 +111,7 @@ impl ChildReaper {
                     str::from_utf8(&output.stderr).context("convert stderr to utf8")?,
                 )?;
             }
+            token.cancel();
```

Comment:
this token.cancel() can be skipped due to the previous line. Do you think it should be fixed?

Reply:
this is before we've registered the child, so there isn't any entity that will call the cancel. I can add a comment